mscore/shortcut.cpp

```diff
@@ -3741,6 +3741,16 @@ Shortcut Shortcut::_sc[] = {
          Icons::mail_ICON,
          Qt::ApplicationShortcut
          },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY,
+         "edit-header-footer",
+         QT_TRANSLATE_NOOP("action","Edit Header/Footer…"),
```

Comment:
not sure the ellipsis is needed here?

Reply:
Should I remove it? I was just maintaining the pattern with rest of the 3 options.

Reply:
Oh, OK. Then leave it in ;-)